src/SDKs/Billing/Billing.Tests/Billing.Tests.csproj

```diff
@@ -5,7 +5,7 @@
     <Description>Billing.Tests Class Library</Description>
     <Authors>Microsoft Corporation</Authors>
     <AssemblyName>Billing.Tests</AssemblyName>
-    <VersionPrefix>2.0.0-preview</VersionPrefix>    
+    <VersionPrefix>2.1.0-preview</VersionPrefix>    
```

Comment:
You don't need to update the package version for test project since they are never published. Also consider simply setting it to `2.0.0` instead

Reply:
Done